.infrastructure/ingress/ingress.yml

```diff
@@ -0,0 +1,18 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+ name: todoapp-ingress
+ annotations:
+   nginx.ingress.kubernetes.io/rewrite-target: /
+spec:
+ rules:
+ - http:
+     paths:
+     - pathType: Prefix
+       path: /
+       backend:
+         service:
+           name: todoapp-service
+           port:
+             number: 80
+             
```

Comment:
EoF didn't display for reason that last line in not empty 
![image](https://github.com/user-attachments/assets/27d340c0-d35d-4697-959c-d95b050337b1)